kubetest2-tf/data/k8s-ansible/group_vars/all

```diff
@@ -11,7 +11,7 @@ username: <username>
 password: <password>
 
 ##### Node OS/Package related updates #####
-update_os_packages: false
+update_os_packages: true
```

Comment:
is this for temporary or permanently we want to make this as true?

Reply:
Removing the task from `update-node-os` role will rule out your dependency on this `update_os_packages` variable.